2020/01/23 16:13:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.99.1, server: localhost, request: "GET /edit-account HTTP/1.0", upstream: "fastcgi://172.23.0.2:9000", host: "176.151.18.182"
2020/01/23 16:13:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.99.1, server: localhost, request: "GET /edit-account HTTP/1.0", upstream: "fastcgi://172.23.0.2:9000", host: "176.151.18.182"
2020/01/23 16:13:48 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.99.1, server: localhost, request: "GET /favicon.ico HTTP/1.0", upstream: "fastcgi://172.23.0.2:9000", host: "176.151.18.182", referrer: "http://176.151.18.182/edit-account"
2020/01/23 16:13:48 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.99.1, server: localhost, request: "GET /favicon.ico HTTP/1.0", upstream: "fastcgi://172.23.0.2:9000", host: "176.151.18.182", referrer: "http://176.151.18.182/edit-account"
2020/01/24 09:54:10 [emerg] 1#1: directive "try_files" is not terminated by ";" in /etc/nginx/nginx.conf:30
2020/01/24 09:56:01 [emerg] 1#1: directive "try_files" is not terminated by ";" in /etc/nginx/nginx.conf:30
2020/01/24 09:57:15 [emerg] 1#1: unexpected "}" in /etc/nginx/nginx.conf:31
2020/01/24 10:12:22 [emerg] 1#1: unexpected "}" in /etc/nginx/nginx.conf:31
2020/01/24 10:12:56 [error] 6#6: *3 open() "/app/favicon.ico" failed (2: No such file or directory), client: 192.168.99.1, server: localhost, request: "GET /favicon.ico HTTP/1.0", host: "176.151.18.182", referrer: "http://176.151.18.182/"
2020/01/24 10:13:33 [emerg] 1#1: unexpected "}" in /etc/nginx/nginx.conf:31
2020/01/24 10:17:20 [emerg] 1#1: unexpected "}" in /etc/nginx/nginx.conf:31
2020/01/24 10:32:07 [error] 6#6: *1 open() "/app/edit-account" failed (2: No such file or directory), client: 192.168.99.1, server: localhost, request: "GET /edit-account HTTP/1.0", host: "176.151.18.182"
2020/01/24 10:38:02 [error] 6#6: *1 open() "/app/edit-account" failed (2: No such file or directory), client: 192.168.99.1, server: localhost, request: "GET /edit-account HTTP/1.0", host: "176.151.18.182"
2020/01/24 10:51:39 [error] 6#6: *1 open() "/app/explore" failed (2: No such file or directory), client: 192.168.99.1, server: localhost, request: "GET /explore HTTP/1.0", host: "176.151.18.182"
2020/01/24 10:52:48 [error] 6#6: *9 open() "/app/explore" failed (2: No such file or directory), client: 192.168.99.1, server: localhost, request: "GET /explore HTTP/1.0", host: "176.151.18.182"
2020/01/24 10:53:04 [error] 6#6: *10 open() "/app/explore" failed (2: No such file or directory), client: 192.168.99.1, server: localhost, request: "GET /explore HTTP/1.0", host: "176.151.18.182"
2020/01/24 10:53:05 [error] 6#6: *11 open() "/app/explore" failed (2: No such file or directory), client: 192.168.99.1, server: localhost, request: "GET /explore HTTP/1.0", host: "176.151.18.182"
2020/01/24 10:53:05 [error] 6#6: *12 open() "/app/explore" failed (2: No such file or directory), client: 192.168.99.1, server: localhost, request: "GET /explore HTTP/1.0", host: "176.151.18.182"
2020/01/24 10:53:06 [error] 6#6: *13 open() "/app/explore" failed (2: No such file or directory), client: 192.168.99.1, server: localhost, request: "GET /explore HTTP/1.0", host: "176.151.18.182"
